web/tests/lynxkite.ts

```diff
@@ -474,7 +473,7 @@ class PlotState extends PopupBase {
     const heights = await this.barHeights()
     await expect(heights.length).toEqual(expected.length);
     for (let i = 0; i < heights.length; ++i) {
-      expect(heights[i]).toBeCloseTo(expected[i]);
+      expect(heights[i]).toBeCloseTo(expected[i], 0);
```

Comment:
Can you please update or delete the comment above? There's no Jenkins and this is not 1% accurate. Thanks!